client/lib/plans/constants.js

```diff
@@ -1288,15 +1291,17 @@ export const FEATURES_LIST = {
 			}
 
 			return i18n.translate(
-				'Get a free custom domain name (example.blog) with this plan ' +
-					'to use for your website. Does not apply to premium domains.'
+				'Get a free .blog domain for one year. Premium domains not included. Your domain will renew at its regular price.'
 			);
 		},
 	},
 
 	[ FEATURE_CUSTOM_DOMAIN ]: {
 		getSlug: () => FEATURE_CUSTOM_DOMAIN,
-		getTitle: () => i18n.translate( 'Custom Domain Name' ),
+		getTitle: () =>
+			i18n.translate( 'Free Domain for One Year', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **2** times:
`translate( 'Free Domain for One Year' )` *ES Score: 12*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

:information_source: This string is probably long enough that it doesn't need a **context**. Are you sure you don't want to use a translator comment instead?